Packs/ThreatMiner/Integrations/ThreatMiner/Threatminer_test.py

```diff
@@ -0,0 +1,20 @@
+import demistomock as demisto
+from ThreatMiner import get_dbot_score_report
+from CommonServerPython import DBotScoreReliability
+
+DBOT_SCORES = {
+    'Reliability': 'C - Fairly reliable',
+    'Vendor': 'ThreatMiner',
+    'Indicator': 'CA978112CA1BBDCAFAC231B39A23DC4DA786EFF8147C4E72B9807785AFEE48BB',
+    'Score': 0,
+    'Type': 'File'
+}
+
+
+def test_reliability_in_dbot(mocker):
```

Comment:
please conform to the when/given/then format

Reply:
done.